universal-login-example/src/App.tsx

```diff
@@ -2,7 +2,8 @@ import React from 'react';
 import Web3 from 'web3';
 import {ULWeb3Provider} from '@universal-login/web3';
 
-const web3 = new Web3(new ULWeb3Provider(new Web3.providers.HttpProvider('https://rinkeby.infura.io')));
+const universalLogin = new ULWeb3Provider(new Web3.providers.HttpProvider('https://rinkeby.infura.io'));
```

Comment:
We should probably parametrize that with either env or sth else.
By default, it should work on dev environment. e.g.

```ts
const httpProviderUrl = ENV.HTTP_PROVIDER_URL || 'https://rinkeby.infura.io';
const universalLogin = new ULWeb3Provider(new Web3.providers.HttpProvider(httpProviderUrl));
```
Alternatively:
```ts
const httpProviderUrlFromUrl = new URL(window.location.href).searchParams.get("httpProviderUrl");
const httpProviderUrl = httpProviderUrlFromUrl || 'https://rinkeby.infura.io';
```

Reply:
Yep, already done in the next PR